docs/registration.adoc

```diff
@@ -0,0 +1,125 @@
+:toc: left
+:toclevels: 3
+:sectanchors: true
+:sectids: true
+:source-highlighter: rouge
+:icons: font
+
+= Registration
+
+== Stake
+
+Stake T tokens to the Staking Provider account according to the instructions
+received from your Staking Provider or 
+the link:https://docs.threshold.network/guides/migrating-legacy-stakes[Threshold Network Staking Guide].
+
+== Authorize Applications
+
+Authorize the Random Beacon and TBTC applications.
+
+// TODO: Add instruction
+
+[#register-operator]
+== Register Operator
+
+Once the tokens are staked the one controlling the Staking Provider account has to
+register an operator for the Random Beacon and TBTC applications.
+
+If a Key File for the Staking Provider account is available the registration
+can be performed from the <<register-operator-client>>.
+
+Another way is registration from the <<register-operator-web>>.
+
+NOTE: For the Keep Legacy stakes the Staking Provider account and Operator account are
+the same.
+
```

Comment:
I think it's worth noting this can be done only one time and that the operator can not be replaced later.

Reply:
454d4542700aabe6aa99367e1cf39a3017777ebd